backend/src/controllers/auth/logout.ts

```diff
@@ -11,6 +11,12 @@ const logout = async (
     const { id: userId } = res.locals;
     await clearTokenProvider(userId);
     res.clearCookie('refreshToken');
+
+    const accessToken = req.headers.authorization?.split(' ')[1];
```

Comment:
Let's finally extract this to separate function, for example `extractAccessTokenValue`.

Reply:
done